portal_objects/workflows/workflow_manta_add_confidence.json

```diff
@@ -0,0 +1,74 @@
+{
+    "accession": "GAPWFR8V3KN3",
+    "app_name": "workflow_manta_add_confidence",
+    "app_version": "VERSION",
+    "arguments": [
+        {
+            "argument_format": "vcf",
+            "argument_type": "Input file",
+            "workflow_argument_name": "input_vcf"
+        },
+        {
+            "argument_format": "vcf_gz",
+            "argument_type": "Output processed file",
+            "workflow_argument_name": "vcf_sv_confidence"
```

Comment:
here we need to specify the secondary files as well, you can probably copy it from some other workflow